src/gameComponents/__tests__/Areas.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { shallowMount, Wrapper } from '@vue/test-utils'
+import Areas from '@/gameComponents/Areas.vue'
+import Vue from 'vue'
+
+const baseMocks = {
+  $store: {
+    state: {
+      scoreStore: {
+        score: 0,
+        lastStyle: 'excellent'
+      },
+      arrowStore: {
+        arrowsData:
+          [{
+            id: 1,
+            direction: 'left-arrow'
+          }]
+      }
+    }
+  }
+}
+
+let wrapper: Wrapper<Areas>
+let vm: Areas | any
+
+describe('Areas.vue', () => {
+  beforeEach(() => {
+    wrapper = shallowMount(Areas, {
+      mocks: baseMocks
+    })
+    vm = wrapper.vm
+  })
+
+  it('animationStyle must consist necessary style', () => {
+    expect(vm.animationStyle).toEqual({
+      good: '',
+      excellent: 'excellent-animation'
+    })
+  })
+
+  it('emit should exist and return necessary obj', async () => {
+    vm.$emit('calculate-positions')
+    await Vue.nextTick()
+    expect(wrapper.emitted('calculate-positions')).toHaveLength(2)
+    expect(wrapper.emitted('calculate-positions')).toMatchObject([
+      [{
+        bottomExcellentArea: 0,
+        bottomGoodArea: 0,
+        topExcellentArea: 0,
+        topGoodArea: 0
+      }],
+      []
+    ])
+  })
+})
+
+describe('Areas.vue', () => {
```

Comment:
why are here 2 `describe`s with the same name? I'd rather have one general `describe` and another `describe` (or some `describe`s)  inside the general one.